bookShop/src/app/state/books.actions.ts

```diff
@@ -0,0 +1,14 @@
+import { createAction, props } from '@ngrx/store';
+import { IBook } from '../shared/models/BookModel';
+
+export const addBook = createAction('[Book List] Add Book', props<IBook>());
```

Comment:
Let's make a `enum` for the type of your actions and use them everywhere.

Reply:
Fixed